hedera-node/hedera-config/src/main/java/module-info.java

```diff
@@ -12,7 +12,9 @@
     requires transitive com.hedera.pbj.runtime;
     requires transitive com.swirlds.config.api;
     requires transitive com.swirlds.config.extensions;
+    requires transitive org.hyperledger.besu.datatypes;
```

Comment:
I think we should try to avoid these dependencies in our config module.

Reply:
Hey, thanks for the review! Can you elaborate on why we should avoid this dependency, please?
 This is needed to support the EVM Address type in the `permittedDelegateCallers`.

Reply:
The bundling between our App and Besu needs to be loose. One never knows what the future brings, and we may be forced to create a version that uses a different VM or includes multiple VMs. To make this possible, we must ensure that as little code as possible depends on Besu and that it is only in a few well-defined areas. I believe so far, only the smart contract module depends on Besu.

I believe in this case, we have a few options:

1. We add the dependency. I would hate that, but this may be the best option if everything else is too much work right now.
2. We move `AddressTypeConverter` and `ContractsConfig` to the smart contracts module. Initially, the plan was to move all `Config`-classes to their specific service module anyway. But we were unsure because having all the configs in one place is also nice. But moving the configs to the code they configure seems cleaner and would allow the definition of specific types. Maybe this is a good time to start moving the `Config`-classes. 🤔
3. We create our own class that stores the value and add a converter to `Address` in the smart contract module.

I had a fourth idea that I forgot. 🙂 Maybe it comes back, and I will add it here.

Reply:
Thanks for the explanation. 🙂 I'll keep that in mind for future work regarding Besu and similar dependencies. I agree with @tinker-michaelj that implementing this using Set<Long> might be the easiest approach, so I went ahead and made the changes